charts/cosigned/templates/webhook/deployment_webhook.yaml

```diff
@@ -48,6 +48,9 @@ spec:
 {{- end }}
 {{- end }}
         args:
+        {{- if semverCompare ">= 1.8-0" .Chart.AppVersion }}
```

Comment:
This is needed to not break the test as cosigned fails to start up when unknown flag is configured and the `--webhook-name` flag will be available in the next release, which should be 1.8.0

Reply:
Yes, I was thinking to pause this PR until we get a new release probably during the next week.

Reply:
Sounds like a plan to me. I can drop the commit with this change later.

Reply:
cc @cpanato any thoughts ?